app/views/events/show.html.erb

```diff
@@ -1,6 +1,18 @@
 <h1><%= @event.name %></h1>
-<subtitle><%= date_range(@event) %></subtitle>
-<p role="doc-subtitle">
+<p role="doc-subtitle"><%= date_range(@event) %></p>
+<% if current_user&.profile&.attending? @event %>
+  <p>Attending</p>
+<% elsif current_user %>
+  <%= button_to "Attend",
+    event_attendees_path(
+      event_attendee: {
+                        profile_id: current_user.profile,
+                        event_id: @event.id
+                      }
+    ),
+    action: "create" %>
+<% end %>
```

Comment:
This makes it a little hard to scan for me. What do you think of something like
```suggestion
<% if current_user %>
<% if current_user.profile&.attending? @event %>
  <p>Attending</p>
<% else %>
  <%= button_to "Attend",
    event_attendees_path(
      event_attendee: {
                        profile_id: current_user.profile,
                        event_id: @event.id
                      }
    ),
    action: "create" %>
<% end %>
<% end %>
```

Reply:
Have the high-level check for `current_user` shows that this logic is related to when the user is logged in. Then, we have the sub check for "are you attending, or not"

Reply:
I like this better than nested ifs, but I agree that it's harder to read. Maybe it could be a helper?